code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -165,6 +170,8 @@
 		energy_released += (FIRE_HYDROGEN_ENERGY_RELEASED * burned_fuel)
 		cached_results["fire"] += burned_fuel * 10
 
+	if(location && prob(10) && burned_fuel > TRITIUM_MINIMUM_RADIATION_ENERGY)
+		radiation_pulse(location, max_range = sqrt(energy_released / FIRE_HYDROGEN_ENERGY_RELEASED) / 2, threshold = 10 * INVERSE(10 + energy_released / FIRE_HYDROGEN_ENERGY_RELEASED), chance = 50)
```

Comment:
Does this cap out? if so at what

Reply:
It has no limit. Is this a problem?

Reply:
@Mothblocks please add tick checks, also advise

Reply:
I have utterly no clue what the range on either that `max_range` or `threshold` are, but max ranges definitely should be capped because no matter what, being irradiated from something very far away just sucks.

Reply:
I'll cap it at 30 when I have the time to get on.

Reply:
What's the range of threshold? It's not obvious to me at all, I want to know that is correct as well.

Reply:
Starts at 1, and converges to 0 the higher the energy is released.

Reply:
Added min check.

Reply:
Could be okay